LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>I'm confused why this is necessary? Just a quick comment would be great</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>yes, this is creating the output from the model instance:</p><p>instance_non_linear.w_i_f.pprint() gives you:</p><pre class="ql-syntax" spellcheck="false">w_i_f : Size=10, Index=w_i_f_index
    Key    : Lower : Value                  : Upper : Fixed : Stale : Domain
    (0, 0) :     0 : -9.988716347647046e-09 :     1 : False :  True :  Reals
    (0, 1) :     0 :     0.5421267530639352 :     1 : False :  True :  Reals
    (0, 2) :     0 :     0.8402967294914276 :     1 : False :  True :  Reals
    (0, 3) :     0 : -9.979664917225373e-09 :     1 : False :  True :  Reals
    (0, 4) :     0 : -9.710092820626022e-09 :     1 : False :  True :  Reals
    (1, 0) :     0 :      0.290246540374087 :     1 : False :  True :  Reals
    (1, 1) :     0 :     0.9122425255472254 :     1 : False :  True :  Reals
    (1, 2) :     0 :     0.2890856799734783 :     1 : False :  True :  Reals
    (1, 3) :     0 : -9.969750471384377e-09 :     1 : False :  True :  Reals
    (1, 4) :     0 : -9.985112854439647e-09 :     1 : False :  True :  Reals
</pre><p>so we are basically just fetching the calculated values</p>